internal/pkg/manifest/validate_env.go

```diff
@@ -49,11 +49,8 @@ func (e EnvironmentConfig) validate() error {
 	if e.CDNEnabled() {
 		cdnCert := e.CDNConfig.Config.Certificate
 		if e.HTTPConfig.Public.Certificates == nil {
-			if cdnCert != nil {
-				return &errFieldMustBeSpecified{
-					missingField:      "http.public.certificates",
-					conditionalFields: []string{"cdn.certificate"},
-				}
+			if cdnCert != nil && !aws.BoolValue(e.CDNConfig.Config.TerminateTLS) {
+				return errors.New(`"cdn.terminate_tls" must be true if "cdn.certificate" is set without "http.public.certificates"`)
```

Comment:
What if they use default Copilot alias and specify `cdn.certificate` for that? But I guess in that scenario they can just remove `cdn.certificate` and it should still work.

Reply:
i think in that case yeah, they should just remove `cdn.certificate` since we manage a cert for them.